README.md

```diff
@@ -59,6 +60,36 @@ Stacks has implemented visual regression testing with [Backstop](https://github.
 
 Individual routes to test are found in [backstop.json](/backstop.json)
 
+## Releasing a new version of Stacks
+Stacks uses [Semantic Versioning](https://semver.org/), is distributed via [npm](https://www.npmjs.com/package/@stackoverflow/stacks), and publishes [release notes on Github](https://github.com/StackExchange/Stacks/releases). Follow the steps below to release a new version of Stacks.
+
+### Bump the version number
+```sh
+npm version [major | minor | patch]
+```
+
+### Push the new tag
+```sh
+git push && git push --tags
+```
+
+### Create release notes [on Github](https://github.com/StackExchange/Stacks/releases/new)
+
+1. Visit https://github.com/StackExchange/Stacks/releases/new
+1. Choose your new version from the "Choose a tag" dropdown
+1. Click "Generate release notes"
+1. Cleanup and complete the release notes
+    - Prominently mention any breaking changes, if applicable
+    - Include a "What's Changed" section in the release notes
+    - Mention significant bug fixes
+    - Mention new features
+    - Mention significant under-the-hood changes that could impact consumers
+
+### Ship your newly created version to [npm](https://www.npmjs.com/package/@stackoverflow/stacks)
```

Comment:
I guess this command will require the user to be logged in to be executed. Perhaps it might be worth mentioning that.

Reply:
The CLI prompts the user to log in if needed (usually, it is because there's a max of 5 minutes for "don't ask me to verify" when publishing), so I feel like that doesn't need to be called out. I'm happy to add a mention though if you have strong feelings here.

Reply:
No strong feelings :)